### PR TITLE
Added the ability to toggle dark mode in the float-menu web component

### LINKED ISSUE #81: [React] Add dark mode to web component
Screenshot 2022-10-26 at 8 55 21 PM

Since the `float-menu` popover is a web component, this is not straightforward.

So the approach should be to pass `isDark` var to `float-menu`, and conditionally have dark mode styles

Follow the reference article - [`Medium`](https://medium.com/technofunnel/creating-passing-data-to-html-custom-elements-using-attributes-bfd9aa759fd4)

### PR DESCRIPTION
**Pull request by simonpankovski**
Fixes #81

### CODE REVIEW

**Comment by anup-a** [comment]
Excellent. Thanks.

Can you please address a couple of comments I left?

**Comment by simonpankovski** [comment]
@anup-a Yes, its not a problem. Could you just point me to the comments you left as I can't seem to find them on my own?

**Comment by anup-a** [comment]
Can you check now? @simonpankovski

**Comment by simonpankovski** [comment]
Yep, all good now. I'll resolve them soon :)

File: lib/webcomponent/float-menu.js [hunk 1]
```diff
@@ -2,6 +2,30 @@ const template = document.createElement('template')
 
 template.innerHTML = `
     <style>
+      .dark-theme {
+        background-color: rgba(14, 20, 27, 1);
```
[the remark this answers is not in the document]
Comment (simonpankovski) [reply lib/webcomponent/float-menu.js]: Updated the value, although the new value doesn't match the rest of the theme. Is it okay like this? 

image

File: lib/webcomponent/float-menu.js [hunk 2]
```diff
@@ -143,6 +161,17 @@ class FloatMenu extends HTMLElement {
...
+  toggleFloatMenuDarkMode(){
+    const homeIcon = this.shadowRoot.querySelector('.home-icon')
+    const iconsDiv = this.shadowRoot.querySelector('.icons')
+    homeIcon.classList.toggle('dark-theme')
+    homeIcon.classList.toggle('light-theme')
+    iconsDiv.classList.toggle('dark-theme')
+    iconsDiv.classList.toggle('light-theme')
+  }
+  attributeChangedCallback(name, oldValue, newValue) {
+    this.toggleFloatMenuDarkMode()
```
[the remark this answers is not in the document]
Comment (simonpankovski) [reply lib/webcomponent/float-menu.js]: Updated the function's parameter using the name attribute.

File: lib/webcomponent/float-menu.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (anup-a) [reply lib/webcomponent/float-menu.js]: Yeah its fine. Previous color was not contrasting enough with background and seemed to overlapping on mobile devices.

**Comment by anup-a** [comment]
Looks good. Thanks 👍.

Merged

**Pull request merged**